neutrinordp/xrdp-neutrinordp.h

```diff
@@ -216,5 +216,6 @@ struct mod
     struct bitmap_item bitmap_cache[4][4096];
     struct brush_item brush_cache[64];
     struct pointer_item pointer_cache[32];
-
+    char pamusername[255];
+    int pid;
```

Comment:
The structure `struct mod` is used to store module persistent data. Since the process id doesn't change, it's not necessary to store it here.

Reply:
Thanks
I'll follow your suggestion.

Reply:
Modified by f33a381.